helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl

```diff
@@ -193,13 +193,6 @@ kubeScheduler:
 
 grafana:
   enabled: false
-  forceDeployDashboards: true
-  defaultDashboardsTimezone: ""
-  sidecar:
-    dashboards:
-      multicluster:
-        global:
-          enabled: true
```

Comment:
Why are these removed? Are they by default now or something?

Reply:
to disable the kube-prometheus-stack dashboards we need `forceDeployDashboards: false` (default)
`defaultDashboardsTimezone` and `grafana.sidecar.dashboards.multicluster.global.enabled` are necessary only if we use the kube-prometheus-stack dashboards. As we disabled them, I removed this lines.

Reply:
Makes sense